build_tools/cmake/iree_bytecode_module.cmake

```diff
@@ -104,4 +105,22 @@ function(iree_bytecode_module)
       "${_TESTONLY_ARG}"
     )
   endif()
+
+  if(DEFINED _RULE_C_OUTPUT AND _RULE_C_OUTPUT)
```

Comment:
```suggestion
  if(_RULE_C_OUTPUT)
```

should be sufficient, now that `C_OUTPUT` is an `<options>` argument.

Reply:
Done. Thanks for catching this.